apps/andi/lib/andi/input_schemas/input_converter.ex

```diff
@@ -308,6 +308,17 @@ defmodule Andi.InputSchemas.InputConverter do
     end)
   end
 
+  defp add_dataset_ids(schema, dataset_ids, parent_bread_crumb \\ "") do
```

Comment:
do we still need the add_dataset_id func in this module if we add the dataset_ids?

Reply:
Yes, it's still used in another function.